.env.example

```diff
@@ -5,4 +5,5 @@ NEXT_PUBLIC_CLOSED_LOCATIONS=""
 NEXT_PUBLIC_RECAP_CLOSED_LOCATIONS=""
 NEXT_PUBLIC_NON_RECAP_CLOSED_LOCATIONS=""
 ADOBE_EMBED_URL=https://assets.adobedtm.com/1a9376472d37/ddf1bedfe52e/launch-4eefcc91c90e.min.js
-SHEP_API=http://[fqdn]/api/v0.1
\ No newline at end of file
+SHEP_API=http://[fqdn]/api/v0.1
+LOGIN_BASE_URL=https://[fqdn]
```

Comment:
Add this to the `ENVIRONMENTVARS.md` file. I think the existing description from DFE is still applicable: https://github.com/NYPL/discovery-front-end/blob/development/ENVIRONMENTVARS.md?plain=1#L53